src/global/tracking/tracking.cc

```diff
@@ -166,9 +186,27 @@ void InitPlugin(JApplication* app) {
                                                               },
                                                               app));
 
+  app->Add(new JOmniFactoryGeneratorT<ActsTrajectoriesMerger_factory>(
+      "CentralB0CKFActsTrajectories",
+      {
+          "CentralCKFActsTrajectories",
+          "B0TrackerCKFActsTrajectories",
```

Comment:
better move this down to after where B0TrackerCKFActsTrajectories is defined.

Reply:
I agree, pushed a commit to implement this.